lib/chat/query.py

```diff
@@ -0,0 +1,44 @@
+import graphene
+
+from bson import ObjectId
+
+from .model import Chat
+from .type import ChatType
+
+from lib.chatMessage.model import ChatMessage
+from lib.match.model import Match
+from lib.adoptable.model import Adoptable
+
+
+class Query:
+    chat = graphene.Field(ChatType, adoptable_id=graphene.String())
+
+    def resolve_chat(self, info, adoptable_id):
+        token = getattr(info.context, "token", None)
+
+        if token is None:
+            return {}
+
+        chat = Chat.objects(adoptable_id=ObjectId(adoptable_id)).all()
+
+        if len(chat) != 1:
+            return {}
```

Comment:
Same as above:
Return something more usefull like an empty list since chat which is a list of objects